tree/dataframe/test/datasource_csv.cxx

```diff
@@ -400,9 +401,11 @@ TEST(RCsvDS, NaNTypeIndentification)
 
 TEST(RCsvDS, NanWarningChecks)
 {
+#ifdef R__USE_IMT
    ROOT::DisableImplicitMT(); // to allow usage of display
+#endif
```

Comment:
This was previously necessary because the IMT section was written before this section. Now when this test is being hit the implict MT has not been activated yet so this call is redundant